experimental/opencv-text-recognition/text_recognition.py

```diff
@@ -110,24 +124,20 @@ def clean_output(text):
 
 for pathToImg in tqdm(args['image']):
 
+    print("[INFO] Processing image: {}".format(pathToImg))
```

Comment:
For readability sake let's use f-strings

`print(f'[INFO] Processing image: {pathToImg}')`